packages/app/src/settings/device/NotificationSettings.vue

```diff
@@ -157,7 +157,14 @@ const switches = [
   },
 ]
 
-const listRef = ref(props.gql.localSettings.preferences.notifyWhenRunCompletes)
+const listRef = ref()
+
+// allow for gql value to load when navigating straight here from EnableNotificationsBanner
+watchEffect(() => {
```

Comment:
IIRC there's a `vueuse/core` function that effectively does this a bit more succinctly, but this gets the job done so I'm fine with it